mint-auditor/src/db/models/audited_mint.rs

```diff
@@ -503,12 +541,9 @@ mod tests {
         insert_mint_tx_from_deposit(&deposit, &conn, &mut rng);
 
         config.tokens[0].token_id = TokenId::from(123);
-
-        assert!(matches!(
-            AuditedMint::try_match_deposit_with_mint(&deposit, &config, &conn),
-            Err(Error::DepositAndMintMismatch(_))
-        ));
-
+        let result = AuditedMint::try_match_deposit_with_mint(&deposit, &config, &conn);
```

Comment:
Can you please explain why a bunch of tests got changed? I don't see why they should be producing different results all of the sudden.

Reply:
They're producing different results because they're throwing different errors after the changes I made. Instead of getting caught inside verify, noting that the tokenid is wrong when you look up this mint and does not correspond to the deposit, you fail to find the mint at all.